<?xml version="1.0" ?>
<launch>
  <node name="RiCTraffic" output="screen" pkg="ric_board" type="Start.py"/>
  <param command="$(find xacro)/xacro.py '$(find ric_description)/komodo/komodo_no_arm_no_plugins2.xacro'" name="robot_description"/>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher"/>
  <include file="$(find openni2_launch)/launch/openni2.launch">
    <arg name="camera" value="Asus_Camera"/>
  </include>
  <node name="front_cam" output="screen" pkg="usb_cam" respawn="true" type="usb_cam_node">
    <param name="video_device" value="/dev/video0"/>
    <param name="camera_frame_id" value="front_cam"/>
    <param name="pixel_format" value="yuyv"/>
    <param name="image_width" value="640"/>
    <param name="image_height" value="480"/>
  </node>
  <node name="laser" output="screen" pkg="hokuyo_node" type="hokuyo_node">
    <param name="port" value="/dev/Hokuyo"/>
    <param name="frame_id" value="Laser_link"/>
  </node>
  <rosparam command="load" file="$(find ric_board)/config/komodo.yaml"/>
</launch>
